app/components/ui/CalculatorInput.tsx

```diff
@@ -280,41 +280,41 @@ const CalculatorInput = () => {
                     >
                       <div className="flex items-center space-x-2">
                         <RadioGroupItem
-                          value="0.015"
+                          value={MAINTENANCE_LEVELS[0].toString()}
                           id="radio-option-low"
                           className="radio-button-style"
                         />
                         <Label
                           htmlFor="radio-option-low"
                           className="radio-label-style"
                         >
-                          Low (1.5%)
+                          Low ({MAINTENANCE_LEVELS[0] * 100}%)
                         </Label>
                       </div>
                       <div className="flex items-center space-x-2">
                         <RadioGroupItem
-                          value="0.02"
+                          value={MAINTENANCE_LEVELS[1].toString()}
                           id="radio-option-medium"
                           className="radio-button-style"
                         />
                         <Label
                           htmlFor="radio-option-medium"
                           className="radio-label-style"
                         >
-                          Medium (2%)
+                          Medium ({MAINTENANCE_LEVELS[1] * 100}%)
                         </Label>
                       </div>
                       <div className="flex items-center space-x-2">
                         <RadioGroupItem
-                          value="0.0375"
+                          value={MAINTENANCE_LEVELS[2].toString()}
                           id="radio-option-high"
                           className="radio-button-style"
                         />
                         <Label
                           htmlFor="radio-option-high"
                           className="radio-label-style"
                         >
-                          High (3.75%)
+                          High ({MAINTENANCE_LEVELS[2] * 100}%)
```

Comment:
Maybe we should model these as an object, not an array.

```suggestion
                          High ({MAINTENANCE_LEVEL["high"] * 100}%)
```

Reply:
Good shout--just started on that before realising it will make my next PR more complicated 😅 so jut going to make a card for this and move on for now!